custom_components/ha_washdata/frontend.py

```
@@ -266,6 +269,61 @@ async def _on_lovelace_loaded(event: Event) -> None:
 
 
+async def _do_register_static_paths(hass: HomeAssistant, src: Path) -> bool:
+    """Register all WashData panel static HTTP paths.  Returns True on success."""
+    try:
+        # Panel JS (primary asset — must be available before the sidebar fires).
+        try:
+            from homeassistant.components.http import StaticPathConfig  # pylint: disable=import-outside-toplevel
+
+            if hasattr(hass.http, "async_register_static_paths"):
+                await hass.http.async_register_static_paths(
+                    [StaticPathConfig(PANEL_JS_URL, str(src), True)]
```

Ruff (0.15.21)

[warning] Boolean positional value in function call

(FBT003)